Mathlib/Data/Polynomial/AlgebraMap.lean

```diff
@@ -178,6 +178,9 @@ theorem adjoin_X : Algebra.adjoin R ({X} : Set R[X]) = ⊤ := by
 set_option linter.uppercaseLean3 false in
 #align polynomial.adjoin_X Polynomial.adjoin_X
 
+theorem mem_adjoin_X (f : R[X]) : f ∈ Algebra.adjoin R ({X} : Set R[X]) := by
+  simp only [adjoin_X, Algebra.mem_top]
```

Comment:
I'm not sure this lemma is useful, given that it follows trivially from `adjoin_X`